lib/wallaby/stateless_query.ex

```diff
@@ -0,0 +1,278 @@
+defmodule Wallaby.StatelessQuery do
+  @moduledoc ~S"""
+  Provides the query DSL.
+
+  Queries are used to locate and retrieve DOM elements. The standard method for
+  querying is css selectors:
+
+  ```
+  visit("/page.html")
+  |> find("#main-page .dashboard")
+  ```
+
+  If more complex querying is needed then its possible to use XPath:
+
+  ```
+  find(page, {:xpath, "//input"})
```

Comment:
module doc mentions `find` all the time alhough it doesn't seem to be defines here anymore, or well more narrow scoped methods like `css`/`xpath` which I like better than passing a tuple to find either way :)